### ISSUE TITLE
Very first draft

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience-article#6]
Hi all, 

I've uploaded a very first and incomplete draft but I won't have much time to work further on it until 2 to 3 weeks. If anyone feel to work on it...

**Comment by khinsen** [comment]
This draft looks very good to me, being essentially a fleshed-out version of the outline in #4.

Before improving details, I think we should think about two questions:
1. What is the central message of the article?
2. Who is our main audience?

Some candidates for the central message:
- Advertising/increasing visibility: This is our new ReScience journal, wouldn't you want to publish something there?
- The reproducibility problem in computer-aided research: overview of the problem and proposed solutions/tools, followed by a summary of our own approach.
- The value of publishing replications: Here's how reviewing replications in ReScience has worked out in practice, and the benefits we believe it brings to science.

I suspect we all hope for ReScience getting more exposure through this article, but then this is not the role of peer-reviewed articles, so it cannot be the central message in my opinion. The other two candidates are valid choices for a peer-reviewed paper. But we should be careful to focus on one or the other, otherwise referees will justly criticize the paper for lack of focus.

My personal favorite is the last one.

**Comment by oliviaguest** [comment]
I just have read through the 1st and (just glanced at the) 2nd sections. I have just made a PR for some small typos/spelling mistakes in the intro (see: https://github.com/ReScience/ReScience-article/pull/10 - which I just saw has been accepted! Thanks @rougier! 😄 ).

More broadly, I think the English needs reformatting and help both along the lines of rephrasing conceptually and grammatically. This will inevitably bring the size down (which is always useful) and clarify the arguments and raise questions (which are also both useful). I don't think this is that much work per se, but given there are many authors such rephrasing endeavours require more thought and compromise. 

Does anybody have any objections to me reading it all over soon and proposing some more changes? I don't know what the rules are for authorship – have I missed an important relevant thread?

**Comment by rougier** [comment]
Good for me if you can correct my (bad) english !

For the authorship, I've sent a mail to all editors, reviewers (that have reviewed) and published authors. Among these, I took the name of those who declared some interest by watching the thread. The order is pretty much random at the moment. And we may have more authors in the end. I'll send another email I think.

**Comment by rougier** [comment]
@khinsen I agree with you that we cannot write an article solely for advertising the journal and your third point (_The value of publishing replications: Here's how reviewing replications in ReScience has worked out in practice, and the benefits we believe it brings to science._) makes perfect sense to me.

At this point, I'm not sure how to structure the paper around this message. Most certainly in the introduction and the conclusion but ideas welcome.

**Comment by rougier** [comment]
Is someone working on the article right now ?

**Comment by oliviaguest** [comment]
I am not but I still need to go through it. It's on my todo list.

Tangentially, has there been a discussion of where to publish this that I have missed?

**Comment by rougier** [comment]
Yes, it's issue #3. If you have more suggestion, that would be great.
